src/scss/components/_tabs.scss

```diff
@@ -1,33 +1,36 @@
 .nav-tabs {
-  border-bottom: none;
+  &--product,
+  &--real-results {
```

Comment:
why do we need this nesting?

Reply:
This is what I mentioned before, the tabs css for homepage is affecting the tab in product page, so I asked pebi to adjust it.